plugins/obs-qsv11/QSV_Encoder_Internal.cpp

```diff
@@ -718,9 +799,9 @@ mfxStatus QSV_Encoder_Internal::Encode(uint64_t ts, uint8_t *pDataY,
 #endif
 
 	while (MFX_ERR_NOT_FOUND == nTaskIdx || MFX_ERR_NOT_FOUND == nSurfIdx) {
-		// No more free tasks or surfaces, need to sync
-		sts = m_session.SyncOperation(
-			m_pTaskPool[m_nFirstSyncTask].syncp, 60000);
+		//No more free tasks or surfaces, need to sync
+		MFXVideoCORE_SyncOperation(
```

Comment:
Why are we switching from the C++ implementation?

Reply:
m_session changed types from MFX_Video_Session to mfxSession and only MFX_Video_Session had the C++ interface.